libraries/exporters/src/main/java/com/siemens/sw360/exporter/ProjectExporter.java

```diff
@@ -91,53 +113,92 @@ public int getColumns() {
         }
 
         @Override
-        public List<String> makeRow(Project project) {
-            List<String> row = new ArrayList<>(getColumns());
+        public SubTable makeRows(Project project) {
+            return extendedByReleases
```

Comment:
All DocumentExporters are subclasses of ExcelExporter. Only the ProjectExporter and ComponentExporter share the methods makeRowForProject/ComponentOnly vs. makeRowWith Releases. We could introduce another layer that is generic in the document type - but I would prefer not to do so.